angular-workspace/projects/ni/nimble-angular/src/thirdparty/directives/radio_control_value_accessor.ts

```diff
@@ -0,0 +1,222 @@
+/**
+ * Copied from https://github.com/angular/angular/blob/e9b5dac9ecaa86914079a37176985c628c403aba/packages/forms/src/directives/radio_control_value_accessor.ts
+ * with the following modifications:
+ * - Commented out part of RadioControlValueAccessor._checkName because ngDevMode is undefined
+ * - Commented out now-unused throwNameError() function
+ * - Removed now-unused import for RuntimeErrorCode
+ */
+
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive, ElementRef, forwardRef, Injectable, Injector, Input, NgModule, OnDestroy, OnInit, Renderer2, ɵRuntimeError as RuntimeError} from '@angular/core';
+
+import {BuiltInControlValueAccessor, ControlValueAccessor, NG_VALUE_ACCESSOR} from './control_value_accessor';
+import {NgControl} from './ng_control';
+
+export const RADIO_VALUE_ACCESSOR: any = {
+  provide: NG_VALUE_ACCESSOR,
+  useExisting: forwardRef(() => RadioControlValueAccessor),
+  multi: true
+};
+
+/* Nimble: Commenting because unused
```

Comment:
nit: I like having the custom prefix to distinguish our comments. Maybe we can do `[Nimble]` to make it stand out a bit more. Whichever format is used it should be mentioned in the thirdparty/README.md that the comment convention is used.

Reply:
Done